README.md

```diff
@@ -1,22 +1,78 @@
-# FFC-Grants-Eligibility-Checker
+# Form Builder
 
-**webpack-cli**
-Run `npm run build` to bundle application
+This project is a flexible and easy-to-use **Form Builder** designed to simplify the process of creating, customizing, and managing eligibility checkers. It leverages **Hapi.js** for the backend framework, with strong code standards enforced using **eslint** and **prettier**. The project is developed with **Node.js** (v20.x and above).
 
-## Local Setup
+## Features
+- Dynamic form creation
+- Field validation and customization
```

Comment:
There's a few Americanisations in here. Are we fussed?

Reply:
I'm fussed and appalled that I put that in there 😂